about.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+  />
+  <script
+  src="https://kit.fontawesome.com/6c59d03160.js"
+  crossorigin="anonymous"
+></script>
+    <title>Capstone Project About Pages</title>
+</head>
+<body>
+  <header>
+    <!-- Nav-bar -->
+    <div class="container">
+      <div class="header_nav">
+        <a href="https://www.facebook.com/"
+          ><i class="fa-brands fa-facebook-f"></i
+        ></a>
+        <a href="https://twitter.com/"
+          ><i class="fa-brands fa-twitter"></i
+        ></a>
+        <a href="#">English</a>
+        <a href="#">My Pages</a>
+        <a href="#">Logout</a>
+      </div>
+    </div>
+    <div class="nav">
+      <div id="hamburger">
+        <div></div>
+      </div>
+    </div>
+    <div id="overlay">
+          <a href="./index.html" class="home-link">Home</a>
+          <a href="#third-section">Program</a>
+          <a href="#news">News</a>
+          <a href="#container">Join</a>
+          <a href="./index.html">Home</a>
+          <a href="#">
+            <span class="cc-campaign">Robo-Campaign</span>
+          </a>
+    </div>
+    <a href="#" class="logo"
+      ><img src="./images/Capture-logo.JPG" alt="nav-logo"
+       /></a>
```

Comment:
- [ ] For good tags semantics I suggest you delimitate your header section correctly and remove the **closed header tag** (`</header>`) at the bottom of the page, Line 188,(as all the content of the page is not part of the header section), then put it at it expected place.